src/index.html

```diff
@@ -2,11 +2,382 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin=" "
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body id="page-top">
+    <header class="header">
+      <a href="https://mickhakk.github.io/layout_catalog/" class="header__logo">
+        <img
+        src="./images/logo.png"
+        alt="logo"
+        class="logo"
+        >
```

Comment:
Spacing + Alt
```suggestion
        <img
          src="./images/logo.png"
          alt="Moyo logo"
          class="logo"
        >
```

Reply:
> Spacing + Alt

Formatting is still not fixed, and in all other places too
- Keep your attributes correctly formatted
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/d5a69ae0-120d-42d1-863c-122a203f530f)